alembic_migration/mysql/env.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+
+from alembic_migration.migration_basesettings import settings
+from tdp.core.models.base_model import BaseModel
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Model's MetaData object here for 'autogenerate' support.
+target_metadata = BaseModel.metadata
+
+# Set the the database_dsn if not already set.
+if config.get_main_option("sqlalchemy.url") == None:
```

Comment:
You do not need to get `sqlalchemy.url` in the `alembic.ini`, we will never put this property in the `alembic.ini`

Reply:
The reason I put this code is that it is not necessarily configured in the `alembic.ini`. You can set the `sqlalchemy.url` where ever you want and if I set it for instance in the tdp `init` command with `config.set_main_option("sqlalchemy.url", <database-dsn>)` it will take it from there. If nothing is set anywhere then I take what is written in the`.env` file.

Reply:
However, I will delete it and if we need it later for the integration we can put it back again.